sympy/series/tests/test_limits.py

```diff
@@ -870,4 +870,4 @@ def test_issue_19766():
 def test_issue_16708():
     m, vi = symbols('m vi', positive=True)
     B, ti, d = symbols('B ti d')
-    assert limit((B*ti*vi - sqrt(m)*sqrt(-2*B*d*vi + m*(vi)**2) + m*vi)/(B*vi), B, 0) == (d + ti*vi)/vi
+    assert limit((B*ti*vi - sqrt(m)*sqrt(-2*B*d*vi + m*(vi)**2) + m*vi)/(B*vi), B, 0) == (d + ti*vi)/vi
```

Comment:
It looks like you've deleted the newline at the end of this file. It needs to be added back

Reply:
ok, done

Reply:
well, but Travis CI still not accepting the code.

Reply:
The tests are still running

Reply:
Do, I have to make some more changes in the code for this particular PR?